bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTConfig.kt

```diff
@@ -101,6 +103,11 @@ data class BKTConfig internal constructor(
       return this
     }
 
+    fun sdkVersion(version: String): Builder {
+      this.sdkVersion = version
+      return this
+    }
```

Comment:
```suggestion
    // Sets the SDK version explicitly.
    // IMPORTANT: This option is intended for internal use only.
    // It should NOT be set by developers directly integrating this SDK. 
    // Use this option ONLY when another SDK acts as a proxy and wraps this native SDK.
    // In such cases, set this value to the version of the proxy SDK.
    fun sdkVersion(version: String): Builder {
      this.sdkVersion = version
      return this
    }
```

Reply:
I think I will use the name `proxySDKVersion`. WDYT? @cre8ivejp 
```suggestion
    fun proxySDKVersion(version: String): Builder {
      this.sdkVersion = version
      return this
    }
```

Reply:
Maybe `wrapperSDKVersion` would be better? Instead of using the term proxy, wrapper would be more intuitive?
WDYT?

Reply:
Yes, it improves clarity for internal devs.

Reply:
@nnnkkk7, WDYT? I know you merged it as `sdkVersion` in the Go SDK, but including the comments would be more intuitive to avoid misunderstanding.

Reply:
Looks good!
I will also add comments like this!

Reply:
@nnnkkk7, thanks!.
Please also rename the function as follows `wrapperSDKVersion`.

@duyhungtnn, let's proceed with the changes in the comments.